### ISSUE TITLE
[css-multicol] Something went wrong with the linking or code block?

### ISSUE THREAD

**Issue by praveenpuglia** [w3c/csswg-drafts#1850]
Here - https://www.w3.org/TR/2017/WD-css-multicol-1-20171005/#example-3694c0ea
& the example after that. 

![image](https://user-images.githubusercontent.com/313363/31238139-955951c0-aa16-11e7-854c-349b8d2ffc48.png)

**Comment by bert-github** [comment]
Somehow the word "or" got marked-up as a code example, instead of the two declarations on either side. Nothing serious, all the text is still there, but it doesn't quite look as it should.

Also, the quote marks in the example aren't wrong, but a bit redundant, typographically speaking. When the code is already set on a line of its own, the quotes aren't really needed. And when the code ends with a newline, you get a closing quote at the start of the line, which looks unusual. That newline is not actually needed, we can remove it. But, given that the quotes are added by the style sheet, we probably need a slightly smarter style sheet anyway.

**Comment by rachelandrew** [comment]
I've updated the editor's draft to use `highlight="css"` instead of `class="css"` in order that these will use the [Pygments syntax highlighting](https://tabatkins.github.io/bikeshed/#syntax-highlighting).

I'm updating the examples anyway to use clearer images, so I'll make sure all these chunks of code are properly highlighted at the same time. Thanks for the report.

**Comment by tabatkins** [comment]
Heh, there are two separate markup bugs in this screenshot.

The first is that Bikeshed doesn't track autolink shortcuts across linebreaks (being conservative for safety), and in the source file there's a linebreak in the middle of the "position: fixed" text. So its starting `''` isn't recognized, its ending `''` is instead interpreted as the start of a css autolink covering the "or", and the `''` at the end of "position: absolute" isn't recognized. I just fixed this.

The second is that `class=css` is used to style things as css snippets, which are surrounded by single quotes. Putting it on a `pre` has some unfortunate implications. Better is to use either `class=lang-css` or `highlight=css` to indicate that it's CSS code that needs highlighting. Thanks for doing that already, @rachelandrew.

Since both issues are now fixed, I'll close this issue. Thanks for the report, @praveenpuglia!
